app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/AppItems.kt

```diff
@@ -86,6 +91,65 @@ fun AppItem(
   }
 }
 
+@Composable
+fun AppItemUpdates(
+  modifier: Modifier = Modifier,
+  app: App,
+  onClick: () -> Unit,
+  installButton: @Composable RowScope.() -> Unit,
+) {
+  Row(
+    modifier = modifier
+      .clickable(onClick = onClick)
+      .fillMaxWidth()
+      .defaultMinSize(minHeight = 96.dp)
+      .padding(top = 16.dp, bottom = 16.dp),
+    verticalAlignment = Alignment.CenterVertically,
+  ) {
+    Box(
+      contentAlignment = Alignment.TopEnd
+    ) {
+      AppIconWProgress(
+        app = app,
+        contentDescription = null,
+        modifier = Modifier.size(64.dp),
+      )
+    }
+    Column(
+      modifier = Modifier
+        .padding(start = 16.dp, end = 16.dp)
+        .weight(1f),
+      verticalArrangement = Arrangement.SpaceEvenly
+    ) {
+      Text(
+        modifier = Modifier.padding(bottom = if (app.packageName == "com.aptoide.android.aptoidegames") 8.dp else 0.dp),
+        text = buildAnnotatedString {
```

Comment:
There's no need to use an annotated string here if the strings are separated by different text composables